_posts/2020-09-30-hows-the-performance-of-ruby-3.0.0-preview1.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "How's the Performance of Ruby 3.0.0-preview1?"
+date: 2020-09-30 7:00:00
```

Comment:
@noahgibbs I think this is the reason `jekyll serve` is not serving this article. It only processes articles that have happened in the past.

Reply:
For whatever reason, I'm not seeing *anybody's* articles. Jekyll build is putting files in _site, so maybe I should just use a local browser. But "jekyll serve" plus pointing my browser at "localhost:4000" isn't showing anything. I'll try again, possibly with local browsing, but that won't show things like links.